src/hooks/useFirstLogin.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+import { Alert } from 'react-native';
+import { useSelector } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { selectIsFirstSignIn } from '../store/selectors/account';
+import { isBiometricSupported } from '../util/biometric';
+
+export default () => {
+  const isFirstSignIn = useSelector(selectIsFirstSignIn);
+  const navigation = useNavigation<any>();
+
+  useEffect(() => {
+    (async () => {
+      const isBiometricSupport = await isBiometricSupported();
+      if (isBiometricSupport && isFirstSignIn) {
+        Alert.alert(
```

Comment:
Could we implement a custom modal with Telios styling not Alert?

Reply:
I get the reference from our "Save Draft" alert. 
-> But If you wish I can create a Modal as Telios styling and we can use where we need it instead of Alert.